helm/fdb-operator/templates/manager/statefulset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "chart.selectorLabels" . | nindent 6 }}
+  serviceName: controller-manager-service
+  template:
+    metadata:
+      labels:
+        {{- include "chart.labels" . | nindent 8 }}
+    spec:
+      volumes:
+        - name: cert
+          secret:
+            defaultMode: 420
+            secretName: {{ .Values.webhook.secret.name }}
+        - name: fdb-certs
+          secret:
+            secretName: {{ .Values.secret.name }}
+      serviceAccountName: default
+      containers:
+      - command:
+        - /manager
+        image: "{{ .Values.operator.image }}:{{ .Values.operator.tag }}"
+        imagePullPolicy: IfNotPresent
+        name: manager
+        volumeMounts:
+          - mountPath: /tmp/cert
+            name: cert
+            readOnly: true
+          - mountPath: /tmp/fdb-certs
+            name: fdb-certs
+            readOnly: true
+        env:
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: SECRET_NAME
+            value: {{ .Values.secret.name }}
+          - name: DISABLE_SIDECAR_TLS_CHECK
```

Comment:
This shouldn't be necessary outside of local development environments.

Reply:
a comment above addresses this.